### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jamiltz**
This change is [Reviewable](https://reviewable.io/reviews/couchbaselabs/mobile-testkit/603)

### CODE REVIEW

**Comment by sethrosetter** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/couchbaselabs/mobile-testkit/603)*

File: README.md [hunk 1]
```diff
@@ -419,38 +419,37 @@ sync_gateway Tests
 ### sync_gateway Test Dependencies
 ==================================
 
-The sync_gateway tests require targeting different cluster topologies of sync_gateway(s) + Couchbase Server(s). Don't worry! We will set this up for you. There are two options for these cluster nodes. You can use EC2 AWS instances or vms. 
+The sync\_gateway tests require targeting different cluster topologies of sync_gateway(s) + Couchbase Server(s). Don't worry! We will set this up for you. There are two options for these cluster nodes. You can use EC2 AWS instances or vms.
```
Comment (tleyden) [inline README.md]: Why was this changed to `sync\_gateway`?  Which editor are you using?

**Comment by jamiltz** [comment]
Removed slash. I'm using visual studio code.

**Comment by jamiltz** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[README.md, line 422 \[r1\]](https://reviewable.io:443/reviews/couchbaselabs/mobile-testkit/603#-KNl2kPT8ps1WhkgfhB4-r1-422:-KO6I_N5MNCyBeWDErMh:-499146009) ([raw file](https://github.com/couchbaselabs/mobile-testkit/blob/4717e9d77f01ec5afdf3069e87e06a78c6482700/README.md#L422)):*
<details><summary><i>Previously, tleyden (Traun Leyden) wrote…</i></summary>
> Why was this changed to `sync\_gateway`?  Which editor are you using?
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/couchbaselabs/mobile-testkit/603)*
